agent/agent.go

```diff
@@ -448,3 +485,7 @@ func (a *Agent) workAck(m *nats.Msg, accepted bool, msg string) error {
 
 	return nil
 }
+
+func isSandboxed() bool {
+	return strings.ToLower(os.Getenv("NEX_SANDBOX")) != "false"
```

Comment:
what happens if we want to run no-sandbox on a node we can't go set the environment on?  Should the `nex [dev]run` command be telling the agent that it is sandboxed or not?

Reply:
the run command should never be able to dictate whether the node is sandboxing or not, mainly for security reasons. We can add a tag to the node indicating that it's unsafe